applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -22,9 +22,9 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_SCANNING_LOG_LEVEL);
 
-#define SCAN_TRIG_CHECK_MS    K_SECONDS(1)
-#define SCAN_TRIG_TIMEOUT_MS  K_SECONDS(CONFIG_DESKTOP_BLE_SCAN_START_TIMEOUT_S)
-#define SCAN_DURATION_MS      K_SECONDS(CONFIG_DESKTOP_BLE_SCAN_DURATION_S)
+#define SCAN_TRIG_CHECK_MS    (1 * MSEC_PER_SEC)
```

Comment:
Not my code, but it makes me wonder. 
There is an update here from K_SECONDS(1) to (1 * MSEC_PER_SEC)
While in other places something exactly opposite in some e.g.: 
ble_latency.c:22
#define LOW_LATENCY_CHECK_PERIOD_MS	5000 -> K_SECONDS(5)
Shouldn't there be a unified approach?

Reply:
@MarekPorwisz It depends on the use-case, if the constant is used in a kernel function, which accepts `k_timeout_t` argument, you need to use `K_SECONDS()` etc, to convert the timeout value from integer into a structure. 
On the other hand, if the constant used those helper macros but was expected to be an integer, it can no longer be used, hence conversion like in this example. 
The reason for this conversions is that timing in Zephyr used rely on integers only, so it was safe (and actually recommended) to use those macros. Since it was recently reworked to be based on `k_timeout_t`, which is a structure,  and all macros were aligned with this approach, it is no longer possible to use them in all cases.

Reply:
Thanks, forgot that K_SECONDS returns struct.